docs/server/source/drivers-clients/websocket-event-stream-api.rst

```diff
@@ -0,0 +1,105 @@
+The WebSockets Events Stream API
+================================
+
+BigchainDB provides real-time event streams over the WebSockets protocol with
+the Events Stream API.
+
+Connecting to a Events Stream from your application enables a BigchainDB node
+to notify you as events are processed, such as new `validated transactions <#valid-transactions>`_.
+
+
+Determining Support for the Events Stream API
+---------------------------------------------
+
+In practice, it's a good idea to make sure that the node you're connecting with
+has support for the Events Stream API. To do so, send a HTTP GET request to the
+node's :ref:`Root URL <bigchaindb-root-url>` and check that the response
+contains a ``streams_<version>`` property in ``_links``::
+
+    {
+      "_links": {
+         "streams_v1": "ws://example.com:9984/api/v1/streams/"
+      }
+    }
+
+
+Connection Keep Alive
```

Comment:
It's important to note that the WebSockets IETF spec for ping/pong is different from how I've tried to structure it here; theirs is at the *protocol* level while what I've written is supposed to be implemented at the *application* level.

I think the main reason most people implement their own version is because the *protocol* way of handling pings/pongs is usually hidden from the user; I think for a lot of WebSockets implementations, the user is never exposed to the *protocol* ping/pongs (and if they ever happen).